resharper/src/resharper-unity/Rider/UnityPluginDetector.cs

```diff
@@ -17,11 +15,9 @@ public class UnityPluginDetector
         
         private readonly ISolution mySolution;
         private readonly ILogger myLogger;
-        private static readonly string[] ourPluginFilesV180 = {"RiderAssetPostprocessor.cs", "RiderPlugin.cs"};
```

Comment:
Am I right that the last time we used this was pre-release Rider, so there were no stable Rider versions with this plugin variation bundled?

Reply:
yes